autogpt/agent/agent.py

```diff
@@ -11,6 +11,8 @@
 from autogpt.log_cycle.log_cycle import (
     FULL_MESSAGE_HISTORY_FILE_NAME,
     NEXT_ACTION_FILE_NAME,
+    PROMPT_SUPERVISOR_FEEDBACK_FILE_NAME,
+    SUPERVISOR_FEEDBACK_FILE_NAME,
```

Comment:
great ! now please fix the test that fails.
![image](https://user-images.githubusercontent.com/9652976/236830278-b113320c-3f44-4e9f-9275-d0e6ff62e08e.png)